debian/update.sh

```diff
@@ -73,6 +73,7 @@ export KBUILD_BUILD_USER="support"
 export KBUILD_BUILD_HOST="kunbus.com"
 export ARCH
 export CROSS_COMPILE
+# shellcheck disable=SC2016
```

Comment:
Please add `Expressions don't expand in single quotes, use double quotes for that.` to the commit message